matrix-meetings-widget/src/components/meetings/MeetingsCalendar/MeetingCalenderDetails/MeetingCalenderDetailsContent.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2022 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CheckOutlinedIcon from '@mui/icons-material/CheckOutlined';
+import ContentCopyOutlinedIcon from '@mui/icons-material/ContentCopyOutlined';
+import EventRepeatIcon from '@mui/icons-material/EventRepeat';
+import LinkIcon from '@mui/icons-material/Link';
+import TodayIcon from '@mui/icons-material/Today';
+import {
+  Box,
+  DialogContent,
+  Grid,
+  IconButton,
+  Link,
+  Typography,
+} from '@mui/material';
+import { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useCopyToClipboard } from 'react-use';
+import { isRecurringCalendarSourceEntry } from '../../../../lib/utils';
+import { Meeting } from '../../../../reducer/meetingsApi';
+import { fullLongDateFormat } from '../../../common/DateTimePickers';
+import { useMeetingUrl } from '../../MeetingCardShareMeetingContent/useMeetingUrl';
+import { formatRRuleText } from '../../RecurrenceEditor/utils';
+import { MeetingCalenderDetailsParticipants } from './MeetingCalenderDetailsParticipants';
+import { MeetingCalenderDetailsShare } from './MeetingCalenderDetailsShare';
+
+export function MeetingCalenderDetailsContent({
+  meeting,
+  isBigWindow,
+}: {
+  meeting: Meeting;
+  isBigWindow: boolean;
+}) {
+  const { t } = useTranslation();
+  const { url: meetingUrl } = useMeetingUrl(meeting);
+  const [hasCopied, setHasCopied] = useState(false);
+  const [, copyToClipboard] = useCopyToClipboard();
+
+  const handleOnClick = useCallback(() => {
+    copyToClipboard(meetingUrl);
+    setHasCopied(true);
+  }, [meetingUrl, copyToClipboard]);
+
+  const handleOnBlur = useCallback(() => setHasCopied(false), []);
+
+  const recurrence = isRecurringCalendarSourceEntry(meeting.calendarEntries)
+    ? formatRRuleText(meeting.calendarEntries[0].rrule, t)
+    : '';
+
+  return (
+    <DialogContent sx={{ mt: 1 }}>
+      <Grid container spacing={2}>
+        <Grid item xs={isBigWindow ? 6 : 12}>
```

Comment:
MUI provides [breakpoints](https://mui.com/material-ui/customization/breakpoints/) that we should use instead of manually setting this. https://mui.com/material-ui/react-grid/ describes how it works with the `Grid` component.